articles/migrate/tutorial-migrate-vmware-agent.md

```diff
@@ -197,11 +204,19 @@ Finish setting up and registering the replication appliance.
 1. In appliance setup, select **Setup connectivity**.
 2. Select the NIC (by default there's only one NIC) that the replication appliance uses for VM discovery, and to do a push installation of the Mobility service on source machines.
 3. Select the NIC that the replication appliance uses for connectivity with Azure. Then select **Save**. You cannot change this setting after it's configured.
+
+   > [!TIP]
+   > If for some reason you need to change the NIC selection and you have not clicked the **Finalize configuration** button in step 12, you can do so by clearing your browser cookies and restarting the **Configuration Server Management Wizard**.
+  
 4. If the appliance is located behind a proxy server, you need to specify proxy settings.
     - Specify the proxy name as **http://ip-address**, or **http://FQDN**. HTTPS proxy servers aren't supported.
 5. When prompted for the subscription, resource groups, and vault details, add the details that you noted when you downloaded the appliance template.
 6. In **Install third-party software**, accept the license agreement. Select **Download and Install** to install MySQL Server.
 7. Select **Install VMware PowerCLI**. Make sure all browser windows are closed before you do this. Then select **Continue**.
+
+> [!NOTE]
+> In newer versions of the Replication Appliance the **VMware PowerCLI** installation is not required.
+
```

Comment:
```suggestion
   
   > [!NOTE]
   > In newer versions of the Replication Appliance the **VMware PowerCLI** installation is not required.
   
```
indenting the NOTE to prevent step numbering from restarting at 1